README.md

```diff
@@ -102,17 +103,21 @@ When the Jupyter notebook is loaded and the kernel is ready then we can start ex
 
 ![Notebook loaded](doc/source/images/wml-3-notebook-loaded.png)
 
+> **Important**: *Make sure that you stop the kernel of your notebook(s) when you are done, in order to prevent leaking of memory resources!*
```

Comment:
this sounds like a "memory leak" bug.  I'm hoping you meant account quota resource in general and in that case I'd suggest you change "prevent leaking..." to something like "conserve resources."

Reply:
Done.